python_src/src/lamp_py/tableau/jobs/rt_rail.py

```diff
@@ -128,29 +119,35 @@ def parquet_schema(self) -> pyarrow.schema:
                 ("static_trip_id_guess", pyarrow.string()),
                 ("static_start_time", pyarrow.int64()),
                 ("static_stop_count", pyarrow.int64()),
-                ("first_last_station_match", pyarrow.bool_()),
+                ("exact_static_trip_match", pyarrow.bool_()),
                 ("static_version_key", pyarrow.int64()),
                 ("travel_time_seconds", pyarrow.int32()),
                 ("dwell_time_seconds", pyarrow.int32()),
                 ("headway_trunk_seconds", pyarrow.int32()),
                 ("headway_branch_seconds", pyarrow.int32()),
-                ("updated_on", pyarrow.timestamp("us")),
             ]
         )
 
     def create_parquet(self, db_manager: DatabaseManager) -> None:
         create_query = self.table_query % ""
 
+        # this is a fairly wide dataset, so dial back the batch size
+        # to limit memory usage
+        db_batch_size = 1024 * 1024 / 2
+
         if os.path.exists(self.local_parquet_path):
             os.remove(self.local_parquet_path)
 
         db_manager.write_to_parquet(
             select_query=sa.text(create_query),
             write_path=self.local_parquet_path,
             schema=self.parquet_schema,
+            batch_size=db_batch_size,
         )
 
     def update_parquet(self, db_manager: DatabaseManager) -> bool:
+        dataset_batch_size = 1024 * 1024
```

Comment:
why is the update double the create size?

Reply:
In practice, we shouldn't see very large update query sizes, unless the parquet process itself is turned off for awhile, but I'll update this one as well, just to cover our basis.